src/filesystem.rs

```diff
@@ -89,4 +89,9 @@ impl Filesystem
 	{
 		object.remove_from(self)
 	}
-}
+	
+	pub(crate) fn create_file(&self, object: &mut dyn ElementFilesystem) -> Result<(), Error>
+	{
```

Comment:
I'm not sure that it is necessary to do this, Couldn't the `insert` method be a more appropriate solution for this task?